nominations/templates/admin/application_pdf.html

```diff
@@ -0,0 +1,182 @@
+{% extends "easy_pdf/base.html" %}
+
+
+
+{% block content %}
+    {% load static %}
+    <link rel="stylesheet" href="{% static "dist/css/main.min.css" %}?v=122">
```

Comment:
Do we have to manually version the css files with url params? This is not automated?

Reply:
Correct. Ticket to fix here https://trello.com/c/D76LqScs/9-implement-automated-hashing-for-static-assets